BuildScript.Nuke/BaseBuild.Compile.cs

```diff
@@ -30,13 +34,23 @@ namespace Remotion.BuildScript;
 [SuppressMessage("ReSharper", "MemberCanBePrivate.Global")]
 public partial class BaseBuild : NukeBuild
 {
+  private Task<IReadOnlyCollection<Project>>? _sortedProjectList;
+
+  private Task<IReadOnlyCollection<Project>> SortedProjectList
+  {
+    get { return _sortedProjectList ??= GetSortedProjects(Solution.Path); }
+  }
```

Comment:
```suggestion
  private IReadOnlyCollection<Project>? _sortedProjectList;
  private IReadOnlyCollection<Project> SortedProjectList => _sortedProjectList ??= GetSortedProjects(Solution.Path).GetAwaiter().GetResult();
```

This should be safe to do here.